Support/Scripts/ds2-gdb-wrapper.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
Haha, true. I almost forgot about that!

On Thu, Jun 9, 2016 at 5:36 PM, kokarez <notifications@github.com> wrote:

> In Support/Scripts/ds2-gdb-wrapper.sh
> <https://github.com/facebook/ds2/pull/356#discussion_r66545800>:
>
> > @@ -0,0 +1,16 @@
> > +#! /bin/bash
>
> DS2 support freebsd 💃
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/356/files/556ef336524b1f05211a10186222fd68525c5c71..d047a9007eb41771f1c3099bdf3123c65041c990#r66545800>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbIOtV2ftVfzoGEm91g2WcL3KUfDTks5qKLF6gaJpZM4InmxB>
> .
>



-- 
Stephane Sezer